src/Symfony/Component/Config/ConfigCache.php

```diff
@@ -34,10 +35,14 @@ public function __construct(
         string $file,
         private bool $debug,
         ?string $metaFile = null,
+        array $skippedResourceTypes = [],
```

Comment:
Would it be worth adding a bit of PHPDoc for this parameter? The same you added in `SkippingResourceChecker`